dhis-2/dhis-support/dhis-support-system/pom.xml

```diff
@@ -177,8 +177,9 @@
       <artifactId>quick</artifactId>
     </dependency>
     <dependency>
-      <groupId>net.sourceforge.jexcelapi</groupId>
-      <artifactId>jxl</artifactId>
+      <groupId>org.apache.poi</groupId>
+      <artifactId>poi</artifactId>
+      <version>4.1.1</version>
```

Comment:
Let's move the version to the dependency management in root pom, and remove the version from this one.

Reply:
Yeap, done.